Dockerfile.aro

```diff
@@ -2,5 +2,5 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal
 RUN microdnf update && microdnf clean all
```

Comment:
this PR is missing SRE golden rule style metrics for the gateway service, make sure these are documented in a VSTS story
- CPU/Memory/Disk on VMSS
- Bandwidth used, #connections , #connections per second in gateway itself
- etc.